packages/browser/src/extensions/surveys.tsx

```diff
@@ -907,6 +909,10 @@ function getPopoverPosition(
     position: SurveyPosition = SurveyPosition.Right,
     surveyWidgetType?: SurveyWidgetType
 ) {
+    if (type === SurveyType.ExternalSurvey) {
```

Comment:
similar question to https://github.com/PostHog/posthog/pull/37542#issuecomment-3248144613